front-end/src/Employee/Registration/employee-registration-types.tsx

```diff
@@ -0,0 +1,33 @@
+export interface EmployeeRegistrationFormProps {
+  firstName: string;
+  middleName: string;
+  lastName: string;
+  idNumber: string;
+  dob: string;
+  age: string;
+  telephone: string;
+  email: string;
+  gender: string;
+  kraPin: string;
+  nssf: string;
+  nhif: string;
+  pfNumber: string;
+  salutation: string;
+}
+
+export const formValues: EmployeeRegistrationFormProps = {
+  firstName: '',
+  middleName: '',
+  lastName: '',
+  idNumber: '',
+  dob: '',
+  age: '',
+  telephone: '',
+  email: '',
+  gender: 'Male',
```

Comment:
Why are giving this an initial value of `male` better to leave as empty string

Reply:
I am using radio buttons as input for that field and if I leave it empty, radio buttons does not have props for validation like Text input i.e invalid and invalidText hence making the field open to empty value.
Need advice on how I would tame that.

Reply:
In that case then we need to validate on form submission and have a label to indicate that the field is required

Reply:
Okay, I will work on that.